src/RadworksGovernor.sol

```diff
@@ -0,0 +1,218 @@
+  Governor,
+  GovernorVotesComp,
+  RadworksGovernorTimelockCompound,
+  GovernorSettings
+{
+  struct ProposalVote {
+    uint256 againstVotes;
+    uint256 forVotes;
+    uint256 abstainVotes;
+  }
+
+  /// @notice The address of the RAD token on Ethereum mainnet from which this Governor derives
+  /// delegated voting weight.
+  ERC20VotesComp private constant RAD_TOKEN =
+    ERC20VotesComp(0x31c8EAcBFFdD875c74b94b077895Bd78CF1E64A3);
+
+  /// @notice The address of the existing Radworks DAO Timelock on Ethereum mainnet through
+  /// which this Governor executes transactions.
+  IRadworksTimelock private constant TIMELOCK =
+    IRadworksTimelock(payable(0x8dA8f82d2BbDd896822de723F55D6EdF416130ba));
+
+  /// @notice Human readable name of this Governor.
+  string private constant GOVERNOR_NAME = "Radworks Governor Bravo";
+
+  /// @notice The number of RAD (in "wei") that must participate in a vote for it to meet quorum
+  /// threshold.
+  ///
+  /// TODO: placeholder
+  uint256 private constant QUORUM = 100_000e18; // 100,000 RAD
+
+  /// @param _initialVotingDelay The initial voting delay this Governor will enforce.
+  /// @param _initialVotingPeriod The initial voting period this Governor will enforce.
+  /// @param _initialProposalThreshold The initial number of RAD required to submit
+  /// a proposal this Governor will enforce.
+  constructor(
+    uint256 _initialVotingDelay,
+    uint256 _initialVotingPeriod,
+    uint256 _initialProposalThreshold
+  )
+    GovernorVotesComp(RAD_TOKEN)
+    GovernorSettings(_initialVotingDelay, _initialVotingPeriod, _initialProposalThreshold)
+    RadworksGovernorTimelockCompound(TIMELOCK)
+    Governor(GOVERNOR_NAME)
+  {}
+
+  /// @dev Mapping from proposal ID to vote tallies for that proposal.
+  mapping(uint256 => ProposalVote) private _proposalVotes;
+
+  /// @dev Mapping from proposal ID and address to the weight the address
+  /// has cast on that proposal, e.g. _proposalVotersWeightCast[42][0xBEEF]
+  /// would tell you the number of votes that 0xBEEF has cast on proposal 42.
+  mapping(uint256 => mapping(address => uint256)) private _proposalVotersWeightCast;
+
+  /// @inheritdoc Governor
+  function supportsInterface(bytes4 interfaceId)
+    public
+    view
+    virtual
+    override(Governor, RadworksGovernorTimelockCompound)
+    returns (bool)
+  {
+    return RadworksGovernorTimelockCompound.supportsInterface(interfaceId);
+  }
+
+  /// @inheritdoc IGovernor
+  // solhint-disable-next-line func-name-mixedcase
+  function COUNTING_MODE() public pure override returns (string memory) {
+    return "support=bravo&quorum=bravo";
+  }
+
+  /// @inheritdoc IGovernor
+  function hasVoted(uint256 proposalId, address account)
+    public
+    view
+    virtual
+    override
+    returns (bool)
+  {
+    return _proposalVotersWeightCast[proposalId][account] > 0;
+  }
+
+  /// @inheritdoc Governor
+  function _quorumReached(uint256 proposalId) internal view virtual override returns (bool) {
```

Comment:
Why does this and the `_voteSucceeded` need to be overrided?

Reply:
I believe it needs to be as RadworksGovernor inherits from Governor